### ISSUE TITLE
Implement "publish on release"

### ISSUE THREAD

**Issue by xnt** [xnt/react-something-nice#13]
Publish to NPM when we tag and create a release: https://medium.com/faun/automate-your-npm-publish-with-github-actions-dfe8059645dd

**Comment by hugolesta** [comment]
It seems very interesting for me, May I help you? 

For testing purposes, are you thinking about CI/CD using for example  [travis](https://travis-ci.org/) ?

**Comment by xnt** [comment]
Hi @hugolesta ! If I got it right, this change will require me linking my personal npm account with this repo's settings. That's why I'd like to do it myself.

The CI/CD would definitely be awesome. To get more value though, perhaps we want to add some tests. Is that something you have experience with? Feel free to create an issue and add a harness, if you want, that we can later integrate in Travis or a similar service.

**Comment by hugolesta** [comment]
Hello xnt, it sounds awesome, yep I have experience in automation using travis and other tools, perhaps I could open new issue but first let me think what we can perform to, probably I can build a dummy project using docker and upload to someplace like heroku or something like that. Let me know if you've some suggestion.

**Comment by xnt** [comment]
Nope, sounds awesome. Try creating a separate issue if possible, though.
